content/knowledge-codemagic/build-inputs.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Build inputs
+description: Specify optional inputs that are passed to the workflow when starting a build
+weight: 6
+---
+
+## Overview
+
+Build inputs are customizable parameters you can define within your workflow to make it more adaptable to different scenarios. With build inputs, you can create a single workflow and run it with different configurations by providing the values for inputs when starting a build for the workflow. For example, you can use build inputs to determine whether to build the workflow for test or release purposes or which app flavor to build. This eliminates the need to create multiple similar workflows with specific hardcoded values, making the workflow more reusable and dynamic.
+
+Inputs are workflow-specific and are defined in `codemagic.yaml` under the `inputs` mapping (see the [example](#minimal-example) below). The started workflow receives specified input values in the `inputs` context, i.e. `${{ inputs.inputId }}` is replaced with the value passed to input with identifier `inputId`.
+
+### Minimal example
+
+{{< highlight yaml "style=paraiso-dark">}}
```

Comment:
will be nice to add short intro: "This example configures one input with `ID=name`. The name input is not required and will default to `Codemagic` value. 

- because `name` looks like reserved word, but in reality this is user-defined input ID.
- is it correct example? is it possible to use `inputs` in scripts? I was thinking we need to create environment variable first 🤔